src/actions/preferences.ts

```diff
@@ -102,6 +102,20 @@ export function makeGroupMessageVisibleIfNecessary(channelId: string): ActionFun
     };
 }
 
+export function setCustomStatusInitialisationState(initializationState: string) {
```

Comment:
0/5 at this point, but it might be nice to pass in an object here and have this function be smart enough to stringify it for you. Similarly, you could add a selector to `selectors/entities/preferences` that knows to parse the JSON and return it as an object.

Reply:
Never mind on the second half of that. I see you have selectors that do that on the webapp-side

Reply:
Done.